### DISCUSSION TITLE
How to manually add more keywords like TODO in order to highlight it in the comments?

### DISCUSSION THREAD

**Discussion by ZhichaoDuan** [microsoft/pylance-release#4073] [Enhancement]

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by judej** [comment]
If there is enough upvotes to this, we can ask vscode to support custom keywords. Visual Studio support custom keywords.

**Comment by rchiodo** [comment]
The location where this is specified is here:

https://github.com/microsoft/vscode/blob/2486976f9f480e806576ba1688d6770c5487ed3f/extensions/python/syntaxes/MagicPython.tmLanguage.json#L1986

Theoretically you could manually edit the tmlanguage file on disk that VS code ships.
